trade-finance-manager-api/src/graphql/helpers/amendment.helpers.js

```diff
@@ -64,80 +62,40 @@ const calculateUkefExposure = (facilityValueInGBP, coverPercentage) => {
   return null;
 };
 
-// checks for coverStartDate and returns based on deal-type as stored differently
-const dealTypeCoverStartDate = (facilitySnapshot) => {
-  const { coverStartDate } = facilitySnapshot;
+// returns tfm object from latest amendment if it exists or null
+const findLatestCompletedAmendment = (amendments) => {
+  // array reversed to get the latest amendment
```

Comment:
Can we please put a check to ensure `amendments` is not empty before calling `reverse()` as on an empty array it will throw an error.

Reply:
everything that calls it has a check => facility?.amendments?.length

Reply:
changed to have null check

Reply:
Can we please change it to `!amendments` in case it is null then accessing `.length` will thrown an error.

Reply:
done